src/overlays/actors/ovl_En_Zov/z_en_zov.c

```diff
@@ -60,20 +59,20 @@ static ColliderCylinderInit sCylinderInit = {
     { 20, 40, 0, { 0, 0, 0 } },
 };
 
-static AnimationHeader* D_80BD270C[] = {
-    &object_zov_Anim_00D3EC, &object_zov_Anim_008120, &object_zov_Anim_00B4CC, &object_zov_Anim_00A888,
-    &object_zov_Anim_00C510, &object_zov_Anim_00CAA8, &object_zov_Anim_008120, &object_zov_Anim_00A888,
-    &object_zov_Anim_002B5C, &object_zov_Anim_00418C, &object_zov_Anim_005A6C, &object_zov_Anim_0066A4,
-    &object_zov_Anim_0017D4, &object_zov_Anim_0023F4,
+static AnimationHeader* sAnimationInfo[] = {
```

Comment:
This is an array of just the animations, so this should be named `sAnimations` instead.
You could also add an anim enum if you want, but it could be also be left for a future docs PR

Reply:
ohh so `sAnimationInfo` is for the enum? sorry didnt know lol

Reply:
`sAnimationInfo` is for arrays of the `AnimationInfo` struct.
And don't worry, it can be confusing. I had to search in the repo what we usually do to make sure if it was the right name or not